lib/rbs/unit_test/convertibles.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+module RBS
+  module UnitTest
+    module Convertibles
+      class BlankSlate < BasicObject
+        instance_methods.each do |im|
+          next if %i[__send__ __id__].include? im
```

Comment:
We might want to remove `__id__` from this list, as it doesnt warn when you `undef` it

Reply:
Hmm. Will try it.